src/js/EventEmitter.js

```diff
@@ -0,0 +1,20 @@
+class EventEmitter {
+  constructor() {
+    this.events = {
+      loggedIn: [],
```

Comment:
Maybe we should not hardcode this field?

Reply:
Changed